.github/workflows/helm-test.yml

```diff
@@ -3,7 +3,7 @@ name: Test
 on:
   push:
     branches: [ "main" ]
-  pull_request:
+#  pull_request:
```

Comment:
It was for the moment. Every time I committed one of Beverly's changes, it would kick off another round of testing. I wanted to pause tests until the PR was merged. It's been re-enabled.